pkgs/by-name/me/melos/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildDartApplication,
+}:
+let
+  pname = "melos";
+  version = "6.1.0";
+  src = fetchFromGitHub {
+    owner = "invertase";
+    repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21